pallets/maintenance-mode/src/weights.rs

```diff
@@ -0,0 +1,76 @@
+
+//! Autogenerated weights for `pallet_asset_registry`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-10-04, STEPS: `20`, REPEAT: 10, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! HOSTNAME: `bernardo-benchmarking`, CPU: `AMD EPYC 7B13`
+//! EXECUTION: None, WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
+
+// collected on a c2d-highcpu-8 of Google Cloud Platform
+
+// Executed Command:
+// ./target/release/trappist-collator
+// benchmark
+// pallet
+// --chain
+// dev
+// --pallet
+// pallet_asset_registry
+// --extrinsic
+// *
+// --steps
+// 20
+// --repeat
+// 10
+// --output
+// pallets/asset-registry/src/weights.rs
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use sp_std::marker::PhantomData;
+
+pub trait WeightInfo {
+	fn activate_maintenance_mode() -> Weight;
+	fn deactivate_maintenance_mode() -> Weight;
+}
+
+/// Weight functions for `pallet_asset_registry`.
+pub struct SubstrateWeight<T>(PhantomData<T>);
+impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
+	// Storage: Assets Asset (r:1 w:0)
+	// Storage: AssetRegistry AssetIdMultiLocation (r:1 w:1)
+	// Storage: AssetRegistry AssetMultiLocationId (r:0 w:1)
+	fn activate_maintenance_mode() -> Weight {
+		Weight::from_ref_time(18_710_000)
+			.saturating_add(T::DbWeight::get().reads(2))
+			.saturating_add(T::DbWeight::get().writes(2))
+	}
+	// Storage: AssetRegistry AssetIdMultiLocation (r:1 w:1)
+	// Storage: AssetRegistry AssetMultiLocationId (r:0 w:1)
+	fn deactivate_maintenance_mode() -> Weight {
+		Weight::from_ref_time(16_570_000)
+			.saturating_add(T::DbWeight::get().reads(1))
```

Comment:
Looks suspicious that 2 calls with such a similar behavior have different number of reads.

Reply:
I forgot running the weights and what we had here was just a placeholder. Should be good now.